logging/fb/fluent-bit_config.configmap_opensearch.yaml

```diff
@@ -4,7 +4,7 @@ data:
     [FILTER]
        Name modify
        Match *
-       Set  fb_configMap_version 0.2.18
+       Set  fb_configMap_version 0.2.19_test
```

Comment:
Should this still be 0.2.19_test or does it need to be switched back to 0.2.19?

Reply:
The "_test" is a temporary value we remove during the end-of-sprint clean up prior to publishing a new release.